docs/metrics/exemplars/README.md

```diff
@@ -1,99 +1,154 @@
+
+## Export metrics and traces from the application
+
+Create a new console application and run it:
 
 ```sh
+dotnet new console --output exemplars
+cd exemplars
 dotnet run
 ```
 
-Once the application is running, navigate to
-[http://localhost:5000/weatherforecast]("http://localhost:5000/weatherforecast")
-from a web browser. You may use the following Powershell script to generate load
-to the application.
-
-```powershell
-while($true)
-{
-    Invoke-WebRequest http://localhost:5000/weatherforecast
-    Start-Sleep -Milliseconds 500
-}
+Add reference to [OTLP
+Exporter](../../../src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md):
+
+```sh
+dotnet add package OpenTelemetry.Exporter.OpenTelemetryProtocol
 ```
 
-## Use Exemplars to navigate from Metrics to Traces
+Now copy the code from [Program.cs](./Program.cs) and run the application again.
+The application will start sending metrics to Prometheus and traces to Jaeger.
 
-The application sends metrics (with exemplars), and traces to the OTel
-Collector, which export metrics and traces to Prometheus and Tempo
-respectively.
+The application is configured with trace-based exemplar filter, which enables
+the OpenTelemetry SDK to attach exemplars to metrics:
+
+```csharp
+var meterProvider = Sdk.CreateMeterProviderBuilder()
+    ...
+    .SetExemplarFilter(ExemplarFilterType.TraceBased)
+    ...
+```
 
-Please wait for 2 minutes before continuing so that enough data is generated
-and exported.
+## Use exemplars to navigate from metrics to traces
 
 Open Grafana, select Explore, and select Prometheus as the source. Select the
-metric named "http_server_duration_bucket", and plot the chart. Toggle on the
-"Exemplar" option from the UI and hit refresh.
+metric named "MyFruitCounter_total", and plot the chart. Toggle on the
+"Exemplars" option from the UI and hit refresh.
 
-![Enable Exemplar](https://user-images.githubusercontent.com/16979322/218627781-9886f837-11ae-4d52-94d3-f1821503209c.png)
+![Enable Exemplars](https://github.com/open-telemetry/opentelemetry-dotnet/assets/17327289/fc11fd4b-5601-4728-acdd-5129d75fbc04)
 
 The Exemplars appear as special "diamond shaped dots" along with the metric
-charts in the UI. Select any Exemplar to see the exemplar data, which includes
+charts in the UI. Select any exemplar to see the exemplar data, which includes
 the timestamp when the measurement was recorded, the raw value, and trace
 context when the recording was done. The "trace_id" enables jumping to the
-tracing backed (tempo). Click on the "Query with Tempo" button next to the
-"trace_id" field to open the corresponding `Trace` in Tempo.
+tracing backed (Jaeger in this case). Click on the "Query with Jaeger" button
+next to the "trace_id" field to open the corresponding trace in Jaeger.
 
 ![Navigate to trace with exemplar](https://user-images.githubusercontent.com/16979322/218629999-1d1cd6ba-2385-4683-975a-d4797df8361a.png)
```

Comment:
the screenshot is still Tempo, need an updated one.

Reply:
I want to wait for the other comments to be resolved before updating the screenshot (to avoid redoing the work).

Reply:
Updated.